packages/react/src/chip/Chip.css.ts

```diff
@@ -0,0 +1,117 @@
+import { theme } from "../styles";
+import { createVar, style } from "../vanilla-extract";
+import { type RecipeVariants, recipe } from "../vanilla-extract";
+
+const accentColorVar = createVar();
+const solidAccentColorVar = createVar();
+const subtleAccentColorVar = createVar();
```

Comment:
Do we need this? As there's not much color or tweaking, I think we don't need var.